grafana/DOCS.md

```diff
@@ -139,6 +148,17 @@ Assistant Cloud. This includes embedding Grafana resources with an iframe or
 rendered image inside of a dashboard. For more details see
 [Anonymous login not working, Grafana add-on 3.0.0 #55](https://github.com/hassio-addons/addon-grafana/issues/55).
 
+## Unsigned plugins
+
+Starting with Grafana 7.x, it is mandatory to have plugins signed when running in production mode.
+However, there is an option to allow loading unsigned plugins:
+
+```yaml
+env_vars:
+  - name: GF_PLUGINS_ALLOW_LOADING_UNSIGNED_PLUGINS
+    value: my-plugin-name
+```
```

Comment:
We can make this an option?

Reply:
We can. I'll do the change.

Reply:
Done. Now, you just have to set a property in the plugins declaration.